tools/c7n_terraform/tests_terraform/tf_common.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2020 Cloud Custodian Authors.
```

Comment:
one mechanical thing, we should change the license headers on the new files in this pr to using the current spdx styled headers we're using throughout the codebase.

Reply:
Makes sense, updated c7n_terraform code files to use this copyright header vs the previous style